vendor/vendor.json

```diff
@@ -1819,12 +1819,30 @@
 			"revision": "36cf5b690dcc00ff0f34ff7799209050c3d0c59a",
 			"revisionTime": "2019-09-27T01:04:35Z"
 		},
+		{
+			"checksumSHA1": "NCbMhsHDjvCTNYw10fV7deTC8/4=",
```

Comment:
Could you please use revision `36cf5b690dcc00ff0f34ff7799209050c3d0c59a`? For some reason `github.com/containerd/containerd/...` has different revisions... Or I don't mind if you updated everything under `github.com/containerd/containerd` to the version you are using. There does not seem to be a bit change since August in their repo.